Extension marketplace blurb for schema create/evolve functionality. (#604)
confluentinc/vscode: README.md

Schema definitions can be viewed by expanding the schema subject to see a specific schema version,
then clicking the **View Schema** icon.

##### Create / Evolve

> New or updated schemas and subject-bindings can be made simply by creating or opening an `.avsc`,

Reviewer: ```suggestion
You can create new schemas or update schemas and subject-bindings by creating or opening an `.avsc`,
```

`.proto`, or `.json` file, then using the **Cloud Upload** icon in the upper-right of the buffer
titlebar to begin the process of selecting a schema registry and subject to bind to. You can either
bind to new subjects or provide a revised/evolved schema for an existing subject to establish a new
version.

> We recommend searching the marketplace for extensions to validate your Avro, JSON schema, or

Reviewer: ```suggestion
Search the marketplace for extensions to validate your Avro, JSON schema, or
```
